coreSDK/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -40,11 +39,33 @@ export class AppInsightsCore implements IAppInsightsCore {
         });        
 
         this._extensions = extensions.sort((extA, extB) => {
-            return extA.priority > extB.priority ? -1 : 1;
+            let typeExtA = typeof (<any>extA).processTelemetry;
+            let typeExtB = typeof (<any>extB).processTelemetry;
+            if (extA && typeExtA === 'function' && typeExtB === 'function') {
```

Comment:
the if statement is missing ' && extB &&'

Reply:
Fixed, cleaned up sort implementation